src/main/java/com/mraof/minestuck/entity/underling/BasiliskEntity.java

```diff
@@ -98,6 +99,13 @@ public void die(DamageSource cause)
 		}
 	}
 	
+	@Override
+	protected void onAttackStart()
+	{
+		// Will stop the entity while performing its attack animation
+		this.getNavigation().stop();
+	}
```

Comment:
This is not enough currently to prevent any movement before the end of the animation (which also then can mess with animations), modifying the movement speed via attributes may be required

Reply:
This issue extends to all underlings and consorts, although not all have issues with their animations because of it

Reply:
That would've been a problem before as well, but I could look at it and see if I come up with something that works better.

Reply:
It has been a problem with the changes during geckolib, not necessarily something that needs to be completed during this PR but if you have any suggestions we can resolve this conversation

Reply:
If you add a //TODO I would consider this resolved for now